app/models/conference.rb

```diff
@@ -81,6 +82,21 @@ def user_registered? user
   end
 
   ##
+  # Delete all EventSchedules that are not in the hours range
+  # After the conference has been successfully updated
+  def delete_event_schedules
+    if self.start_hour_changed? || self.end_hour_changed?
+      event_schedules = EventSchedule.select do |e|
```

Comment:
@lagartoflojo I think it would be even cooler if we add a `has_many thought` (using schedules relation for example) relation in program, as it is easier. and with that you can just do here:

`conference.program.event_schedules.select do |event_schedule|`

So, it implies just modifying two lines :wink: Add it in a different commit in this PR please :slightly_smiling_face: